stable/dask-distributed/templates/dask-jupyter-deployment.yaml

```diff
@@ -0,0 +1,70 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "jupyter-fullname" . }}-config
+  labels:
+    heritage: {{.Release.Service | quote }}
+    release: {{.Release.Name | quote }}
```

Comment:
Use consistent formatting everywhere, i. e. space after `{{` and space before `}}`.